addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/handler/SolarEdgeHandler.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge.handler;
+
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.solaredge.config.SolarEdgeConfiguration;
+import org.openhab.binding.solaredge.internal.connector.WebInterface;
+import org.openhab.binding.solaredge.internal.model.Channel;
+
+/**
+ * public interface of the {@link SolarEdgeHandler}
+ *
+ * @author afriese
+ *
+ */
+public interface SolarEdgeHandler extends ThingHandler {
+    /**
+     * Called from {@link SolarEdgeWebInterface#authenticate()} to update
+     * the thing status because updateStatus is protected.
+     *
+     * @param status Bridge status
+     * @param statusDetail Bridge status detail
+     * @param description Bridge status description
+     */
+    public void setStatusInfo(ThingStatus status, ThingStatusDetail statusDetail, String description);
```

Comment:
Interface methods are by definition public therefore the access specifier `public` should be omitted on interface methods.

Reply:
fixed it